sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3091,6 +3120,24 @@ public partial class DiskAccessPatch
         public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct DiskControllerType : System.IEquatable<Azure.ResourceManager.Compute.Models.DiskControllerType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public DiskControllerType(string value) { throw null; }
+        public static Azure.ResourceManager.Compute.Models.DiskControllerType NVMe { get { throw null; } }
```

Comment:
Should we rename this to `Nvme`?

Reply:
it is kind of a special word, see here: https://nvmexpress.org/
therefore I prefer we leave it as is